modules/ROOT/pages/nav-matrix.adoc

```diff
@@ -5,7 +5,7 @@ include::{partialsdir}/attributes.adoc[]
 [discrete]
 == Prerequisites
 
-* [x] You have xref:ROOT:index.adoc[set up AeroGear Mobile Services on OpenShift].
+* [x] You have set up xref:getting-started.adoc[AeroGear Mobile Services] on OpenShift.
```

Comment:
@finp - Should this be linking to the #setting-up-aerogear-mobile-services-on-openshift anchor in the getting started document rather than just linking to the base file... Also wondering if it should be an adoc or a html file that is linked to.

Reply:
@johnfriz  the syntax is correct (adoc rather than html)
but as you suggest the following is better:

xref:getting-started.adoc#setting-up-aerogear-mobile-services-on-openshift [AeroGear Mobile Services]

Reply:
Actually... we have a ticket to remove all the pre-req and next steps sections within getting started, so will this not be getting deleted by some other PR? If so, let's just rip it out now...

Reply:
@johnfriz yes. we have this one. you are right. https://issues.jboss.org/browse/AEROGEAR-3513 @ciaranRoche let's not worry about this change in this pr.